src/dom.js

```diff
@@ -0,0 +1,139 @@
+const container = document.getElementById('container');
+
+const addUnit = (value, unit) => `${value}${unit}`;
+
+const displayErrorMessage = (cityName) => {
+  const resultContainer = document.getElementById('result');
+  resultContainer.innerHTML = '';
+  const city = document.createElement('p');
+  city.classList.add('error');
+  resultContainer.appendChild(city);
+  city.textContent = `Sorry, data for "${cityName}" is unavailable`;
+};
+
+const displayResult = (result, tempUnit) => {
```

Comment:
- [ ] [OPTIONAL] Consider changing also the app background according to the weather result, that and working more on your UI will make this project a good candidate for your portfolio.